### ISSUE TITLE
Queue and trigger frame captures through renderdoccmd.exe

### ISSUE THREAD

**Issue by yifli** [baldurk/renderdoc#939]
## Description

When using renderdoccmd.exe, we can't queue and trigger frame captures just like how it's done via the RenderDoc GUI interface. We can discuss about how to add such a feature. 

## Environment

* RenderDoc build: e.g. v1.0, or 2018-03-12 nightly build.
* Operating System: e.g. Windows 10, Windows 7, Ubuntu Linux, etc
* API: e.g. Vulkan, D3D11, OpenGL, etc

**Comment by baldurk** [comment]
There are a number of issues to figure out here:

1. How do you connect the commands that launch programs with the target control? Ideally you'd want to be able to chain them naturally, how should the target control ID be returned (it's currently printed in a log statement that would need some parsing).
2. Do we have a fire-and-forget style command that connects, does one operation, then disconnects? Or is the intention to have a command that connects and persists until disconnected either executing some script or being interactive - if it's a script, what script, and if it's interactive how is that processed?
3. In addition to 1. above, do we want some way to enumerate available target control programs running on a given IP?
4. Do we want to handle getting information back from a target control connection, like the available captures and active API etc. This could become quite complex but having half a feature is not necessarily better than having no feature, so it should be considered if we're addressing this at all.

In addition, you still haven't explained why you can't use the existing remote server functionality to control the target from a normal desktop. After all the command line interface doesn't allow any meaningful analysis of captures, you will still need to use a UI eventually - why can that UI instance not control the target program instead. If there's a good reason to expose this functionality through the CLI then that's fair enough, but I don't want to duplicate work and add complexity unnecessarily when there may already be a more appropriate implementation.

**Comment by yifli** [comment]
We only run a single benchmark frame, and try to use RenderDoc to capture that frame. So yes,  it's a fire-and-forget style command. We capture the frame on our software GPU model, and our driver is put into a special mode where no commands are sent to GPU in order to save capture time.  We don't need to analyze the capture, the capture file is all we need. 

Let me try again using the GUI to capture.  Probably I did not do that due to some misconception.

**Comment by baldurk** [comment]
Not having heard back on this I assume the GUI is working as expected and can be used to capture.
